src/history/history.service.spec.ts

```diff
@@ -99,7 +105,7 @@ describe('HistoryService', () => {
   describe('createOrUpdateHistoryEntry', () => {
     describe('works', () => {
       it('without an preexisting entry', async () => {
-        const user = new User();
+        const user = {} as User;
```

Comment:
I'm not sure if this will haunt us when we try to enable strict mode.

Reply:
Yes but what is the solution to mitigate that?

Reply:
I think I might have found a solution. I added it to https://github.com/hedgedoc/hedgedoc/issues/835

Reply:
I think we should change this in another PR.

Reply:
After the discussion in #835, I think it's best to leave this as is for now and think about alternatives when we enable strict mode.